test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -440,6 +442,363 @@ spec setup =
             (x * Nothing).to_vector . should_equal nulls
             (x / Nothing).to_vector . should_equal nulls
 
+    Test.group prefix+"Rounding-like operations" <|
+        Test.specify "should name a rounding column correctly" <|
+            table = table_builder [["x", [0.1, 0.9, 3.1, 3.9, -0.1, -0.9, -3.1, -3.9]]]
+            table.at "x" . round . name . should_equal "round([x])"
+
+        test_floatlike type =
+            Test.specify "should allow round on a "+type.to_text+" column" <|
+                table = table_builder [["x", [0.1, 0.9, 3.1, 3.9, -0.1, -0.9, -3.1, -3.9]]]
+                result = table.at "x" . cast type . round
+                result.to_vector.should_equal [0, 1, 3, 4, 0, -1, -3, -4]
+                result.value_type . is_integer . should_be_true
+
+            Test.specify "should allow round on a float column (to >0 decimal places)" <|
+                table = table_builder [["x", [0.51, 0.59, 3.51, 3.59, -0.51, -0.59, -3.51, -3.59]]]
+                result = table.at "x" . cast type . round 1
+                result.to_vector.should_equal [0.5, 0.6, 3.5, 3.6, -0.5, -0.6, -3.5, -3.6]
+                result.value_type . is_integer . should_be_false
+
+            Test.specify "should allow truncate on a "+type.to_text+" column" <|
+                table = table_builder [["x", [0.1, 0.9, 3.1, 3.9, -0.1, -0.9, -3.1, -3.9]]]
+                result = table.at "x" . cast type . truncate
+                result.to_vector.should_equal [0, 0, 3, 3, 0, 0, -3, -3]
+                result.value_type . is_integer . should_be_true
+
+            Test.specify "should allow ceil on a "+type.to_text+" column" <|
+                table = table_builder [["x", [0.1, 0.9, 3.1, 3.9, -0.1, -0.9, -3.1, -3.9]]]
+                result = table.at "x" . cast type . ceil
+                result.to_vector.should_equal [1, 1, 4, 4, 0, 0, -3, -3]
+                result.value_type . is_integer . should_be_true
+
+            Test.specify "should allow floor on a "+type.to_text+" column" <|
+                table = table_builder [["x", [0.1, 0.9, 3.1, 3.9, -0.1, -0.9, -3.1, -3.9]]]
+                result = table.at "x" . cast type . floor
+                result.to_vector.should_equal [0, 0, 3, 3, -1, -1, -4, -4]
+                result.value_type . is_integer . should_be_true
+
+        test_floatlike Value_Type.Float
+        test_floatlike Value_Type.Decimal
+
+        Test.specify "should allow round on an int column" <|
+            table = table_builder [["x", [1, 9, 31, 39, -1, -9, -31, -39]]]
+            result = table.at "x" . round -1
+            result.to_vector.should_equal [0, 10, 30, 40, 0, -10, -30, -40]
+            result.value_type . is_integer . should_be_true
```

Comment:
I'd like to see (possibly for primitive types as well, unless it exists already) an example like:

`123.round 2 == 123` - i.e. what happens if we round an integer to positive decimal places. It should act as identity - but we need to verify this:

```suggestion
        Test.specify "should allow round on an int column" <|
            table = table_builder [["x", [1, 9, 31, 39, -1, -9, -31, -39]]]
            result = table.at "x" . round -1
            result.to_vector.should_equal [0, 10, 30, 40, 0, -10, -30, -40]
            result.value_type . is_integer . should_be_true
            
        Test.specify "should make sure round to positive decimal places on an int column acts as identity" <|
            v = [0, 1, 9, 31, -123]
            table = table_builder [["x", v]]
            result = table.at "x" . round 2
            result.to_vector.should_equal v
            result.value_type . is_integer . should_be_true
```

(especially I'd like to double check that the inferred value type is still `is_integer` as I imagine it's quite possible to make it float by accident)

Reply:
Now I see that the tests below mostly check this, so maybe it is not needed. But the value_type check might still be useful.

Reply:
The test "Returns the correct type" checks the return type of this case, but actually I don't have enough tests checking the case of decimal_places > 0 for integers, so I'll add that.